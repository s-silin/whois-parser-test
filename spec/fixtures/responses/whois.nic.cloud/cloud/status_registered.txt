Domain Name: google.cloud
Registry Domain ID: DDDAA72F4F319486082E1B91922F2483D-ARI
Registrar WHOIS Server:
Registrar URL: www.markmonitor.com
Updated Date: 2017-12-31T10:32:45Z
Creation Date: 2016-01-27T01:19:35Z
Registry Expiry Date: 2019-01-27T01:19:35Z
Registrar: MarkMonitor, Inc.
Registrar IANA ID: 292
Registrar Abuse Contact Email: abusecomplaints@markmonitor.com
Registrar Abuse Contact Phone: +1.2083895740
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Domain Status: clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited
Domain Status: clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited
Registry Registrant ID: CDE1C33C5B2534A75AC903887FF4F6063-ARI
Registrant Name: DNS Admin
Registrant Organization: Google Inc.
Registrant Street: 1600 Amphitheatre Parkway
Registrant Street:
Registrant Street:
Registrant City: Mountain View
Registrant State/Province: CA
Registrant Postal Code: 94043
Registrant Country: US
Registrant Phone: +1.6502530000
Registrant Phone Ext:
Registrant Fax: +1.6502530001
Registrant Fax Ext:
Registrant Email: dns-admin@google.com
Registry Admin ID: CDE1C33C5B2534A75AC903887FF4F6063-ARI
Admin Name: DNS Admin
Admin Organization: Google Inc.
Admin Street: 1600 Amphitheatre Parkway
Admin Street:
Admin Street:
Admin City: Mountain View
Admin State/Province: CA
Admin Postal Code: 94043
Admin Country: US
Admin Phone: +1.6502530000
Admin Phone Ext:
Admin Fax: +1.6502530001
Admin Fax Ext:
Admin Email: dns-admin@google.com
Registry Tech ID: CDE1C33C5B2534A75AC903887FF4F6063-ARI
Tech Name: DNS Admin
Tech Organization: Google Inc.
Tech Street: 1600 Amphitheatre Parkway
Tech Street:
Tech Street:
Tech City: Mountain View
Tech State/Province: CA
Tech Postal Code: 94043
Tech Country: US
Tech Phone: +1.6502530000
Tech Phone Ext:
Tech Fax: +1.6502530001
Tech Fax Ext:
Tech Email: dns-admin@google.com
Name Server: ns2.google.com
Name Server: ns4.google.com
Name Server: ns3.google.com
Name Server: ns1.google.com
DNSSEC: unsigned
URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2018-02-15T15:20:42Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

The Service is provided so that you may look up certain information in relation to domain names that we store in our database.

Use of the Service is subject to our policies, in particular you should familiarise yourself with our Acceptable Use Policy and our Privacy Policy.

The information provided by this Service is 'as is' and we make no guarantee of it its accuracy.

You agree that by your use of the Service you will not use the information provided by us in a way which is:
* inconsistent with any applicable laws,
* inconsistent with any policy issued by us,
* to generate, distribute, or facilitate unsolicited mass email, promotions, advertisings or other solicitations, or
* to enable high volume, automated, electronic processes that apply to the Service.

You acknowledge that:
* a response from the Service that a domain name is 'available', does not guarantee that is able to be registered,
* we may restrict, suspend or terminate your access to the Service at any time, and
* the copying, compilation, repackaging, dissemination or other use of the information provided by the Service is not permitted, without our express written consent.

This information has been prepared and published in order to represent administrative and technical management of the TLD.

We may discontinue or amend any part or the whole of these Terms of Service from time to time at our absolute discretion.
