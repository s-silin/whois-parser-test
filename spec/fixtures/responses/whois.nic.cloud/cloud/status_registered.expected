#domain
  %s == "google.cloud"

#domain_id
  %s == "DDDAA72F4F319486082E1B91922F2483D-ARI"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2016-01-27T01:19:35Z}

#updated_on
  %s %CLASS{time}
  %s %TIME{2017-12-31T10:32:45Z}

#expires_on
  %s %CLASS{time}
  %s %TIME{2019-01-27T01:19:35Z}


#registrar
  %s %CLASS{registrar}
  %s.id           == "292"
  %s.name         == "MarkMonitor, Inc."

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == "CDE1C33C5B2534A75AC903887FF4F6063-ARI"
  %s[0].name          == "DNS Admin"
  %s[0].organization  == "Google Inc."
  %s[0].address       == "1600 Amphitheatre Parkway"
  %s[0].city          == "Mountain View"
  %s[0].zip           == "94043"
  %s[0].state         == "CA"
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.6502530000"
  %s[0].fax           == "+1.6502530001"
  %s[0].email         == "dns-admin@google.com"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "CDE1C33C5B2534A75AC903887FF4F6063-ARI"
  %s[0].name          == "DNS Admin"
  %s[0].organization  == "Google Inc."
  %s[0].address       == "1600 Amphitheatre Parkway"
  %s[0].city          == "Mountain View"
  %s[0].zip           == "94043"
  %s[0].state         == "CA"
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.6502530000"
  %s[0].fax           == "+1.6502530001"
  %s[0].email         == "dns-admin@google.com"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id            == "CDE1C33C5B2534A75AC903887FF4F6063-ARI"
  %s[0].name          == "DNS Admin"
  %s[0].organization  == "Google Inc."
  %s[0].address       == "1600 Amphitheatre Parkway"
  %s[0].city          == "Mountain View"
  %s[0].zip           == "94043"
  %s[0].state         == "CA"
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.6502530000"
  %s[0].fax           == "+1.6502530001"
  %s[0].email         == "dns-admin@google.com"


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns2.google.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns4.google.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.google.com"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns1.google.com"
